test/bin/ci_phase_iso_boot.sh

```diff
@@ -39,6 +41,12 @@ bash -x ./bin/manage_hypervisor_config.sh create
 # repository.
 bash -x ./bin/start_webserver.sh
 
+# Setup a container registry and mirror images.
+if [ "${ENABLE_MIRROR}" ]; then
```

Comment:
```suggestion
if  ${ENABLE_MIRROR}" ; then
```
I think this is how we check boolean vars in bash

Reply:
Done